config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/json/JsonTools.java

```
@@ -78,7 +79,8 @@ public ObjectMapper getObjectMapper(final String type) {
 
     private static ObjectMapper buildObjectMapper(JsonFactory jf) {
         ObjectMapper objectMapper = new ObjectMapper(jf);
-        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        objectMapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

do we want to silently ignore if the user makes some typo in the json/yaml?